fcrepo-auth-webac/src/main/java/org/fcrepo/auth/webac/WebACAuthorizingRealm.java

```diff
@@ -87,7 +89,8 @@
     @Inject
     private ResourceFactory resourceFactory;
 
-    @Inject
+    @Autowired
```

Comment:
Why did this (and other examples) need to change from `@Inject` to `@Autowired`?

Reply:
It seems like Spring does not like resolving the `@Qualifier` for `@Inject`, at least in some circumstances.